app/views/docs/permissions.phtml

```diff
@@ -44,60 +44,62 @@
     </thead>
     <tbody>
     <tr>
-        <td>
-            role:all
-            <p class="margin-top-small text-fade">version >= 0.12</p>
-        </td>
+        <td>Role.any()</td>
         <td>Wildcard permission. Gives anyone read or write access.</td>
     </tr>
     <tr>
-        <td>user:[USER_ID]</td>
+        <td>Role.guests()</td>
+        <td>Access to any guest user with an anonymous session. Logged in users don't have access to this role.</td>
+    </tr>
+    <tr>
+        <td>Role.users()</td>
+        <td>Access to authenticated users.</td>
+    </tr>
+    <tr>
+        <td>Role.user([USER_ID])</td>
         <td>Access to a specific user by his UID.</td>
```

Comment:
Not sure if we use `UID` anywhere. Can we just say `by his ID`? If yes, we might want to update this too: https://github.com/appwrite/docs/blob/c0cd90feeceae9960223fbf98bb3bcc470c39d30/app/views/docs/permissions.phtml#L114